client/src/components/Tab.tsx

```diff
@@ -1,16 +1,44 @@
+import { globalState } from '../store';
 import type { Tab as TabType } from '../config/constants';
-import type { FunctionComponent, ReactElement } from 'react';
+import { useSnapshot } from 'valtio';
+import type { CSSProperties, FunctionComponent, ReactElement } from 'react';
+import { useCallback, useMemo } from 'react';
 
 interface TabProps {
-	activeTab?: string;
-	handleClick: () => void;
+	activeTab?: Pick<TabType, 'name'>;
+	handleClick: (tab: TabType) => void;
 	isFilterTab?: boolean;
 	tab: TabType;
 }
 
-const Tab: FunctionComponent<TabProps> = ({ tab, handleClick, activeTab, isFilterTab = false }): ReactElement => {
+const Tab: FunctionComponent<TabProps> = ({ tab, handleClick, activeTab = 'colorpicker', isFilterTab = false }): ReactElement => {
```

Comment:
Change default activeTab to be an empty string or make a new type for it that only accepts the filter tabs